packages/components/src/styles/interactive/interactive.declaration.ts

```diff
@@ -10,6 +10,10 @@ export default {
       name: 'sd-interactive--disabled',
```

Comment:
this should be in another PR as we use squash-merge in our PRs. This would result in one single commit with changes in 2 different components...

As other teams rely on our changelog, we need to be precise here.

Reply:
@mariohamann @yoezlem any thoughts on this topic?

Reply:
We decided to keep the declaration here since it was missing from #385 and it is only docs.